Oxide.Rust/Libraries/Covalence/RustPlayerManager.cs

```diff
@@ -53,18 +61,29 @@ internal void PlayerJoin(ulong userId, string name)
                 playerData.Add(id, record);
                 allPlayers.Add(id, new RustPlayer(userId, name));
             }
-
-            ProtoStorage.Save(playerData, "oxide.covalence");
         }
 
+        /// <summary>
+        /// When a player connects
```

Comment:
I'd probably go with something like... "Register when a player has connected" instead, as "When a player connects" doesn't really summarize the method.